packages/kbn-esql-ast/src/__tests__/ast_parser.columns.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getAstAndSyntaxErrors as parse } from '../ast_parser';
+
+describe('Column Identifier Expressions', () => {
+  it('can parse un-quoted identifiers', () => {
+    const text = 'ROW a, b.c';
+    const { ast } = parse(text);
+
+    expect(ast).toMatchObject([
+      {
+        type: 'command',
+        args: [
+          {
+            type: 'column',
+            parts: ['a'],
+          },
+          {
+            type: 'column',
+            parts: ['b', 'c'],
+          },
+        ],
+      },
+    ]);
+  });
+
+  it('can parse quoted identifiers', () => {
+    const text = 'ROW `a`, `b`.c, `d`.`👍`.`123``123`';
```

Comment:
Should we add a test for mixed use like: 

> part1.part2.`part``3️⃣`

Reply:
Added!